migration-roadmap/number-field.md

```diff
@@ -0,0 +1,379 @@
+                <sp-icon-chevron75 class="stepper-icon"></sp-icon-chevron75>
+            </sp-infield-button>
+            <sp-infield-button class="button step-down">
+                <sp-icon-chevron75 class="stepper-icon"></sp-icon-chevron75>
+            </sp-infield-button>
+        </span>
+    </div>
+    <div id="sp-help-text-..." aria-live="assertive">
+        <slot name="help-text"></slot>
+    </div>
+    <sp-help-text slot="help-text">Help text</sp-help-text>
+</sp-number-field>
+```
+
+</details>
+
+<details>
+<summary>Legacy (CSS main branch):</summary>
+
+```html
+<div class="spectrum-Stepper">
+    <div class="spectrum-Textfield spectrum-Stepper-textfield">
+        <input
+            type="number"
+            class="spectrum-Textfield-input spectrum-Stepper-input"
+        />
+    </div>
+    <span class="spectrum-Stepper-buttons">
+        <button class="spectrum-InfieldButton spectrum-Stepper-button">
+            <svg class="spectrum-Icon spectrum-UIIcon-ChevronUp100"></svg>
+        </button>
+        <button class="spectrum-InfieldButton spectrum-Stepper-button">
+            <svg class="spectrum-Icon spectrum-UIIcon-ChevronDown100"></svg>
+        </button>
+    </span>
+</div>
+```
+
+</details>
+
+<details>
+<summary>Spectrum 2 (CSS spectrum-two branch):</summary>
+
+```html
+<div class="spectrum-NumberField">
+    <label class="spectrum-FieldLabel spectrum-NumberField-fieldLabel">
+        Label
+    </label>
+    <div class="spectrum-NumberField-inputs">
+        <div class="spectrum-Textfield spectrum-NumberField-textfield">
+            <input
+                type="number"
+                class="spectrum-Textfield-input spectrum-NumberField-input"
+            />
+        </div>
+        <span class="spectrum-NumberField-buttons">
+            <div class="spectrum-InfieldButton-inline">
+                <button
+                    class="spectrum-InfieldButton spectrum-NumberField-button"
+                >
+                    <div class="spectrum-InfieldButton-fill">
+                        <svg
+                            class="spectrum-Icon spectrum-UIIcon-ChevronUp100"
+                        ></svg>
+                    </div>
+                </button>
+                <button
+                    class="spectrum-InfieldButton spectrum-NumberField-button"
+                >
+                    <div class="spectrum-InfieldButton-fill">
+                        <svg
+                            class="spectrum-Icon spectrum-UIIcon-ChevronDown100"
+                        ></svg>
+                    </div>
+                </button>
+            </div>
+        </span>
+    </div>
+    <div class="spectrum-NumberField-helpText">
+        <div class="spectrum-HelpText">
+            <div class="spectrum-HelpText-text">Help text</div>
+        </div>
+    </div>
+</div>
+```
+
+</details>
+
+<details>
+<summary>Diff: Legacy (CSS main) → Spectrum 2 (CSS spectrum-two)</summary>
+
+```diff
```

Comment:
Happy to have any feedback on this numberfield diff... it's really unpleasant to look at, I think, because just about every line changes due to class name changes.